include/dsVideoPort.h

```diff
@@ -364,9 +367,10 @@ dsError_t  dsEnableHDCP(intptr_t handle, bool contentProtect, char *hdcpKey, siz
 /**
  * @brief Indicates whether a video port is DTCP protected.
  * 
- * This function indicates whether the specified video port is configured for DTCP
+ * For source devices, this function indicates whether the specified video port is configured for DTCP
  * content protection. It must return dsERR_OPERATION_NOT_SUPPORTED if DTCP
  * is not supported.
+ * For sink devices, this function returns the authentication status as dsHDCP_STATUS_AUTHENTICATED and returns dsERR_NONE always.
```

Comment:
Why are we talking about HDCP in DTCP api?

Reply:
Removed HDCP related terms from this API